src/features/Apiexplorer/Schema/RecursiveContent/SchemaBodyHeader/__tests__/SchemaBodyHeader.test.tsx

```diff
@@ -74,6 +77,7 @@ describe('SchemaBodyHeader', () => {
         is_open_object
         setIsOpenObject={() => jest.fn()}
         is_stream_types={false}
+        items_type={undefined}
```

Comment:
do we need to pass undefined to items_type?
cant we make items_type as optional in component props?